src/Neo/SmartContract/StorageKey.cs

```diff
@@ -73,7 +91,7 @@ private static void FillHeader(Span<byte> data, int id, byte prefix)
         /// <returns>The <see cref="StorageKey"/> class</returns>
         public static StorageKey Create(int id, byte prefix)
         {
-            var data = new byte[PrefixLength];
+            Span<byte> data = stackalloc byte[PrefixLength];
```

Comment:
this static changes seems good to me, if you can split it in a different pr, we can merge this and discuss the others